src/components/add-review-form/add-review-form.tsx

```diff
@@ -48,12 +47,15 @@ export default function AddReviewForm({ id }: AddReviewFormProps): JSX.Element {
     value.trim().length <= COMMENT_LENGTH.max;
 
   const validationRules = {
-    notEmpty: (value: Comment) => !isEmpty(formData.comment) && !isEmptyRating(value.rating),
+    notEmpty: (value: Comment) =>
+      !isEmpty(formData.comment) && !isEmptyRating(value.rating),
     notTooLong: (value: Comment) => isNotTooLong(value.comment),
-    notToShort: (value: Comment) => isNotShort(value.comment)
+    notToShort: (value: Comment) => isNotShort(value.comment),
   };
 
-  const isValid = Object.keys(validationRules).every((rule) => validationRules[rule as keyof typeof validationRules](formData));
+  const isValid = Object.keys(validationRules).every((rule) =>
+    validationRules[rule as keyof typeof validationRules](formData)
```

Comment:
Лучше вынести повыше отдельным типом 
```suggestion
    validationRules[rule as ValidationRulesKeys](formData)
```